### PR TITLE
Indirect Funding: Test state stub sharing between unit tests and storybook

### PR DESCRIPTION
**Pull request by kerzhner**
Constructing states for testing is hard. Doing so once for unit tests and once for stories is twice as hard.

This PR moves the stub state construction to a scenario for indirect funding.

Fixes https://github.com/magmo/apps/issues/369.

### CODE REVIEW

File: packages/wallet/src/redux/protocols/indirect-funding/player-a/__tests__/stories.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Modal from 'react-modal';
+import { Provider } from 'react-redux';
+import IndirectFundingContainer from '../../../../../containers/indirect-funding/indirect-funding';
```
Comment (tomclose) [inline packages/wallet/src/redux/protocols/indirect-funding/player-a/__tests__/stories.tsx]: Would be nice to move the container into the `protocols/indirect-funding` folder.

File: packages/wallet/src/__stories__/b-dummy-wallet-states.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import * as fundingStates from '../redux/channel-state/funding/state';
 import * as sharedStates from '../redux/channel-state/shared/state';
```
Comment (tomclose) [inline packages/wallet/src/__stories__/b-dummy-wallet-states.ts]: Is the plan to remove this file too at some point?

Comment (kerzhner) [reply packages/wallet/src/__stories__/b-dummy-wallet-states.ts]: Yes, we should remove that file once indirect funding B player scenario is set up.

File: packages/wallet/src/redux/protocols/indirect-funding/player-a/__tests__/stories.tsx (hunk 1 above)
Comment (kerzhner) [reply packages/wallet/src/redux/protocols/indirect-funding/player-a/__tests__/stories.tsx]: Addressed in https://github.com/magmo/apps/pull/379.

**Pull request merged**